initial team practices doc

porta-antiporta opened the pull request.

Commit:
initial commit for comment

fureigh commented on docs/team-practices.md:
```diff
@@ -0,0 +1,116 @@
+
+# Additional Principles
+
+## Coding & Branchin Standards
```
`Branchin` --> `Branching`

Commit:
Copy edits to team practices doc

Made some copy edits to this document, including capitalization updates and shifts from passive > active voice. If you all have any questions or comments, let me know!

Commit:
Merge pull request #3 from 18F/kategarklavs-patch-1

thanks @kategarklavs! merging

fureigh committed:
Fix typo (`Branchin` --> `Branching`)
